src/request/TopLevelApi.js

```diff
@@ -156,6 +157,14 @@ class TopLevelApi extends RequestParser {
                 handler.run();
 
                 TopLevelApi.setLoading(false);
+
+                if (TopLevelApi.getDocumentWidth() > Constants.MIN_WIDTH_FOR_AUTOFOCUS) {
+                    const $page = document.querySelector(window.location.hash);
+                    if ($page && $page.querySelector('.password-box')) {
```

Comment:
Would it be possible to do this instead?
```suggestion
                    if (document.querySelector('.page:target .password-box')) {
```